more detailed example in summary.md (#1514)
rust-lang/mdBook: guide/src/format/summary.md

   ```markdown
   # Title of Part

>    - [Title of thefirst  Chapter](relative/path/to/markdown.md)

Reviewer: ```suggestion
   - [Title of the first  Chapter](relative/path/to/markdown.md)
```

>    - [Title of the first  Chapter](relative/path/to/markdown.md)

Reviewer 2: ```suggestion
   - [Title of the first Chapter](relative/path/to/markdown.md)
```

   - [Title of the second Chapter](relative/path/to/markdown2.md)

>       * [Title of a sub Chapter](relative/path/to/markdown3.md)

Reviewer 2: I would recommend keeping the same style for nested lists, just to avoid any confusion about whether or not that is required. Most markdown styles that I have seen use only one kind of bullet.

```suggestion
      - [Title of a sub Chapter](relative/path/to/markdown3.md)
```
